src/components/search/SearchTypesDirectives.js

```diff
@@ -0,0 +1,395 @@
+(function() {
+  goog.provide('ga_search_type_directives');
+
+  goog.require('ga_browsersniffer_service');
+  goog.require('ga_debounce_service');
+  goog.require('ga_layer_metadata_popup_service');
+  goog.require('ga_map_service');
+  goog.require('ga_marker_overlay_service');
+  goog.require('ga_search_service');
+  goog.require('ga_urlutils_service');
+
+  var originToZoomLevel = {
+    address: 10,
+    parcel: 10,
+    sn25: 8
+  };
+
+  var parseExtent = function(stringBox2D) {
+    var extent = stringBox2D.replace('BOX(', '')
+      .replace(')', '').replace(',', ' ')
+      .split(' ');
+    return $.map(extent, parseFloat);
```

Comment:
Not fully possible, string is "BOX(234.123 123.123, 123.123 123.123)', but will combine.